src/components/pages/UserTimeline.js

```diff
@@ -31,28 +30,41 @@ function UserTimeline() {
 
   return (
     <Page>
-      <Title>
-        <img src={userData.picture} alt="pic" />
-        <h2>{userData.name}'s Posts</h2>
-      </Title>
-      {posts.length ? (
-        posts.map((p, index) => (
-          <Post
-            key={index}
-            postId={p.id}
-            url={p.url}
-            description={p.description}
-            name={p.name}
-            userId={p.userId}
-            picture={p.picture}
-          />
-        ))
-      ) : (
-        <p>{message}</p>
-      )}
-      <Trending />
+      <Navbar />
+      <Wrapper>
+        <Feed>
+          <Title>
+            {/* <Image picture={userPosts.picture} /> */}
```

Comment:
Need to remove the comment

Reply:
Ooooh nooo! I really forgot this one. Thank you. Already removed it